packages/wrangler/src/pages/upload.tsx

```diff
@@ -286,7 +286,7 @@ export const upload = async (
 			}
 		};
 
-		queue.add(() =>
+		await queue.add(() =>
```

Comment:
awaiting this promise will effectively make these upload batches happen in serial. Let's Remove this change
```suggestion
		queue.add(() =>
```